File: backend-example-java/src/main/java/com/example/backend/application/test/TestIocService1.java
package com.example.backend.application.test;

/**
 * 循环依赖测试类1
 */
//@Component
public class TestIocService1 {

     private final TestIocService2 service2;

     public TestIocService1(TestIocService2 service2) {
          this.service2 = service2;
     }
}

File: backend-example-dotnet/src/application/test/TestIocService1.cs
namespace BackendExample.Application.Test
{
    /// <summary>
    /// 循环依赖测试类1
    /// </summary>
    public class TestIocService1
    {
        private readonly TestIocService2 service2;

        public TestIocService1(TestIocService2 service2)
        {
            this.service2 = service2;
        }
    }
}
